linodecli/plugins/obj.py

```diff
@@ -432,6 +432,17 @@ def show_usage(client, args):
             total += obj.size
 
         grand_total += total
+
+        # Coverts bucket size to human readable bytes. 
+        total = float(total)
+        denomination = ["KB", "MB","GB","TB"]
+        for x in denomination:
+            if total > 1024:
+                total = total / 1024 
```

Comment:
This leaves `total` as a float, causing it to blow up when applying padding on line 446.  I would just add `str(total)` to that line to make sure it's always a string.

Reply:
I do convert total to a string in the second condition right before the loop breaks. Would that be sufficient since it's being converted to a string right before it is called to display?